Disable kernel module stripping based on commandline flag: CONFIG_MODULE_DEBUG=y

diff --git a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -379,7 +379,8 @@ KBUILD_CFLAGS_KERNEL :=
 KBUILD_AFLAGS   := -D__ASSEMBLY__
 KBUILD_AFLAGS_MODULE  := -DMODULE
 KBUILD_CFLAGS_MODULE  := -DMODULE
-KBUILD_LDFLAGS_MODULE = -T $(srctree)/scripts/module-common.lds $(if $(CONFIG_PROFILING),,-s)
+KBUILD_LDFLAGS_MODULE = -T $(srctree)/scripts/module-common.lds $(if $(CONFIG_PROFILING)$(CONFIG_MODULE_DEBUG),,-s)
+#CONFIG_MODULE_DEBUG set via command line if needed
 
 # Read KERNELRELEASE from include/config/kernel.release (if it exists)
 KERNELRELEASE = $(shell cat include/config/kernel.release 2> /dev/null)
